fiftyone/core/collections.py

```diff
@@ -947,9 +966,11 @@ def head(self, num_samples=3):
             num_samples (3): the number of samples
 
         Returns:
-            a list of :class:`fiftyone.core.sample.Sample` objects
+            a list of :class:`fiftyone.core.sample.Sample` or
+            :class:`fiftyone.core.sample.SampleView` objects
         """
-        return [s for s in self[:num_samples]]
+        cursor = self._get_first(limit=num_samples)
+        return [self._make_sample(d) for d in cursor]
```

Comment:
we want to return an empty list here, not raise an exception, for backwards compatibility